content_api/tokens/__init__.py

```diff
@@ -45,35 +46,36 @@ def generate_subscriber_token(subscriber, ttl_days=365):
         payload = {'sub': str(subscriber['_id']), 'exp': _timestamp(exp)}
```

Comment:
You already `ex` in `redis.set`

> ex sets an expire flag on key name for ex seconds.

So probably you don't need this `exp` key in payload